packages/dd-trace/src/plugins/util/ci.js

```diff
@@ -81,6 +81,13 @@ function filterSensitiveInfoFromRepository (repositoryUrl) {
   }
 }
 
+function getTestType (frameworkName) {
+  if (frameworkName === 'playwright' || frameworkName === 'cypress') {
+    return 'browser'
+  }
+  return 'test'
+}
```

Comment:
Just to get a bit more consistency on the naming
```suggestion
function getTestTypeFromFramework (testFramework) {
  if (testFramework === 'playwright' || testFramework === 'cypress') {
    return 'browser'
  }
  return 'test'
}
```

Reply:
we can define `getTestTypeFromFramework` within `packages/dd-trace/src/plugins/util/test.js` and it wouldn't need to be exported